.github/workflows/scripts/compare_translations.py

```
@@ -0,0 +1,152 @@
+import json
+import os
+import sys
+from collections import namedtuple
+
+# Named tuple for file and missing translations combination
+FileTranslation = namedtuple(
+    "FileTranslation", ["file", "missing_translations"]
+)
+
+
+def compare_translations(
```

Ruff (0.11.9)

Missing return type annotation for public function `compare_translations`

(ANN201)